toolchain/testing/file_test.cpp

```diff
@@ -194,7 +194,8 @@ auto ToolchainFileTest::GetDefaultArgs() const
   } else if (component_ == "check") {
     args.push_back("--dump-sem-ir");
   } else if (component_ == "lower") {
-    args.push_back("--dump-llvm-ir");
+    args.insert(args.end(),
+                {"--dump-llvm-ir", "--target=x86_64-unknown-linux-gnu"});
```

Comment:
```suggestion
                {"--dump-llvm-ir", "--target=x86_64-linux-gnu"});
```
May as well use the simpler target name here.